src/gui/components/ui/Dialog.tsx

```diff
@@ -67,8 +67,8 @@ export default function FaimsDialog(props: DiagProps) {
           <img style={{objectFit: 'none'}} src={path} />
         ) : (
           <DialogContentText id="alert-dialog-description">
-            To Download attachments and photoes, go to settings page <br />
-            To enable auto sync attachments, go to settings
+            To download attachments and photos, please go to Settings page, and
```

Comment:
"To download attachments and photos, please go to Settings page, and" 
remove 'page'